maps/utk.yml

```diff
@@ -915,7 +903,6 @@ properties:
     range: http://www.w3.org/2001/XMLSchema#string
```

Comment:
Range says string, but value suggests URI.

Reply:
Still looks the same to me?  Line 919 is a URI but 915 says string.

Reply:
Sorry! Must have forgotten to save after.